rpc/common/moc_common_networkcommon.proto

```diff
@@ -51,4 +51,10 @@ message Route {
 	string nextHop = 1;
 	string destinationPrefix = 2;
 	uint32 metric = 3;
+}
+
+message ResourceReference {
+	string Name = 1;
+	string Location = 2;
+	string Group = 3;
```

Comment:
Adding a group here so it is generic for all kinds of resources?

Reply:
yes that was the idea! Nithin mentioned we'll also need this for things like slb in the future